Formula/swimat.rb

```diff
@@ -0,0 +1,24 @@
+class Swimat < Formula
+  desc "Command Line Tool to help format Swift code"
+  homepage "https://github.com/Jintin/Swimat"
+  url "https://github.com/Jintin/Swimat.git",
```

Comment:
Use the release URL instead: https://github.com/Jintin/Swimat/archive/v1.2.1.tar.gz